airlock/views/request.py

```diff
@@ -156,6 +188,47 @@ def request_reject(request, request_id):
 
 @instrument(func_attributes={"release_request": "request_id"})
 @require_http_methods(["POST"])
+def file_approve(request, request_id, path: str):
+    release_request = get_release_request_or_raise(request.user, request_id)
+
+    try:
+        relpath = release_request.get_request_file(path).relpath
+    except bll.FileNotFound:
+        raise Http404()
+
+    try:
+        bll.approve_file(release_request, relpath, request.user)
```

Comment:
This (and the reject equivalent) will need a new audit method - `bll.audit_request_file_approve` or something. The types (AuditEventType) are there already

Reply:
did you see that the bll method [already does that internally](https://github.com/opensafely-core/airlock/blob/main/airlock/business_logic.py#L854-L868)? did you mean that we should pull it out there & call it from views/request instead?

Reply:
Ah - no, I think I was looking at another view where it called an audit method and missed that this was wasn't needed

Reply:
👍